src/components/manage/Blocks/DataFigure/Edit.jsx

```diff
@@ -180,10 +181,14 @@ class Edit extends Component {
   extractTable = async (data) => {
     let arr = [];
     const tableUrl = `${data['@id']}/download.table`;
-    await this.props.getProxiedExternalContent(tableUrl, {
-      headers: { Accept: 'text/html' },
-    });
-    if (this.props.subrequests[tableUrl].error) {
+    if (isInternalURL(tableUrl)) {
+      await getContent(tableUrl, null, tableUrl);
```

Comment:
we should use `this.state.url` here, if we want to render table. But I don't know if its the production issue or due to volto resolving to internal urls.